phpfpm_exporter.go

```diff
@@ -16,6 +16,7 @@ package main
 import (
```

Comment:
Could you run this file through `gofmt`? The imports are not alphabetic right now.